### PR TITLE
Reroute node

### PR DESCRIPTION
**Pull request by magnificus**
looks very ugly but needs some other changes to make it look good

### CODE REVIEW

File: src/nodes/base.ts [hunk 1]
```diff
@@ -321,10 +318,12 @@ export class RangeArray extends PPNode {
...
 export class RandomArray extends PPNode {
+  getColor(): TRgba {
+    return TRgba.fromString(NODE_TYPE_COLOR.INPUT);
+  }
   constructor(name: string, customArgs: CustomArgs) {
     super(name, {
       ...customArgs,
-      color: TRgba.fromString(NODE_TYPE_COLOR.INPUT),
     });
   }
```
Comment (fakob) [inline src/nodes/base.ts:328]: Could the constructor be removed here as well?

File: src/nodes/base.ts [hunk 2]
```diff
@@ -459,10 +457,12 @@ export class If_Else extends PPNode {
...
 export class Comparison extends PPNode {
+  getColor(): TRgba {
+    return TRgba.fromString(NODE_TYPE_COLOR.TRANSFORM);
+  }
   constructor(name: string, customArgs: CustomArgs) {
     super(name, {
       ...customArgs,
-      color: TRgba.fromString(NODE_TYPE_COLOR.TRANSFORM),
     });
   }
```
Comment (fakob) [inline src/nodes/base.ts:468]: Could the constructor be removed here as well?

**Review by fakob — APPROVED** [review]
Added some comments. Looks good.

File: src/nodes/base.ts (hunk 1 above)
Comment (magnificus) [reply src/nodes/base.ts:328]: yes indeed it should be

File: src/nodes/base.ts (hunk 2 above)
Comment (magnificus) [reply src/nodes/base.ts:468]: yes indeed it should be

File: src/nodes/image/shader.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import * as PIXI from 'pixi.js';
 import PPGraph from '../../classes/GraphClass';
 import PPNode from '../../classes/NodeClass';
 import UpdateBehaviourClass from '../../classes/UpdateBehaviourClass';
-import { CustomArgs, TRgba } from '../../utils/interfaces';
+import { CustomArgs, TRgba} from '../../utils/interfaces';
```
[the remark this answers is not in the document]
Comment (magnificus) [reply src/nodes/image/shader.ts:6]: dont know might be autocorrected (?)

**Pull request merged**